external/coreclr/project.json.template

```diff
@@ -3,7 +3,8 @@
     "{TFM}": {
       "dependencies": {
         "Microsoft.NETCore.Platforms": "1.2.0-beta-24721-02",
-        "Microsoft.NETCore.Runtime.CoreCLR": "1.2.0-beta-24728-02"
+        "Microsoft.NETCore.Runtime.CoreCLR": "1.2.0-beta-24728-02",
+        "runtime.native.System.Data.SqlClient.sni": "4.4.0-beta-24814-01",
```

Comment:
how come do we need a runtime-specific package? shouldn't we depend on `System.Data.SqlClient.sni` instead and let the RID add this dependency?